server/integration/testsuite/src/test/java/org/infinispan/server/test/rollingupgrades/HotRodRollingUpgradesIT.java

```diff
@@ -33,30 +33,7 @@
  */
 @RunWith(Arquillian.class)
 @Category({RollingUpgrades.class})
```

Comment:
@mgencur "current" to "current" is a valid test, we do that on Kubernetes for example to migrate data between same versions with incompatible configurations.
Regarding the `older server`, as I mentioned, we could use maven to grab and unpack the older version, wdyt?